bindings/python/manifold3d.cpp

```diff
@@ -140,74 +176,45 @@ PYBIND11_MODULE(manifold3d, m) {
           [](Manifold &self, float x = 0.0f, float y = 0.0f, float z = 0.0f) {
             return self.Translate(glm::vec3(x, y, z));
           },
-          py::arg("x") = 0.0f, py::arg("y") = 0.0f, py::arg("z") = 0.0f,
+          nb::arg("x") = 0.0f, nb::arg("y") = 0.0f, nb::arg("z") = 0.0f,
           "Move this Manifold in space. This operation can be chained. "
           "Transforms are combined and applied lazily."
           "\n\n"
           ":param x: X axis translation. (default 0.0).\n"
           ":param y: Y axis translation. (default 0.0).\n"
           ":param z: Z axis translation. (default 0.0).")
-      .def(
-          "translate",
-          [](Manifold &self, py::array_t<float> &t) {
-            auto t_view = t.unchecked<1>();
-            if (t.ndim() != 1 || t.shape(0) != 3)
-              throw std::runtime_error("Invalid vector shape");
```

Comment:
What happens now if the wrong shape array is input?

Reply:
it will say that there is no appropriate function to call and print the type signature. (Vec3 requires a sequence with length 3, which is checked in the type caster I wrote above)